lib/modules/manager/tekton/extract.ts

```diff
@@ -28,6 +29,7 @@ export function extractPackageFile(
   }
   for (const doc of docs) {
     deps.push(...getDeps(doc));
+    deps.push(...getStepImageDeps(doc));
```

Comment:
please integrate in existing `getDeps` function

Reply:
Good idea! Done